react/features/mobile/react-native-sdk/middleware.js

```diff
@@ -32,6 +35,7 @@ const externalAPIEnabled = isExternalAPIAvailable();
         rnSdkHandlers?.onConferenceFocused && rnSdkHandlers?.onConferenceFocused();
         break;
     case CONFERENCE_JOINED:
+        JitsiMeetOngoingConferenceService.launch();
```

Comment:
I used NativeModules to launch it from here. 
Launch method has context and extradata as params, but I think these two can remain undefined, in this case, right?